src/js/utils/CRONValidatorUtil.js

```diff
@@ -0,0 +1,214 @@
+
+/**
+ * Validator utilities for CRON syntax
+ */
+var CRONValidatorUtil = {
```

Comment:
Let's split this into smaller validation methods (e.g. `isValidMinuteValue`, `isValidMonthValue`) to improve the testability as discussed.